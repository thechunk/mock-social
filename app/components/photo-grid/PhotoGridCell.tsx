import * as React from 'react';
import {PureComponent} from "react";
import TouchableImage from "../common/image/TouchableImage";

export default class PhotoGridCell extends PureComponent<IPhotoGridCell> {
    render() {
        return (
            <TouchableImage onPress={this.props.onPress} style={this.props.style} source={{uri: this.props.url}} />
        );
    }
}