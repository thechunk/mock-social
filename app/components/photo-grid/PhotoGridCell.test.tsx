import * as React from 'react';
import {shallow} from "enzyme";
import PhotoGridCell from "./PhotoGridCell";

describe('PhotoGridCell', () => {
    it('renders correctly', () => {
        expect(shallow(<PhotoGridCell style={{}} url="https://url"/>)).toMatchSnapshot();
    });
});