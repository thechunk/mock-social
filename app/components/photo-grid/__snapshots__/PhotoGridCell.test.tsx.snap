// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoGridCell renders correctly 1`] = `
<TouchableImage
  source={
    Object {
      "uri": "https://url",
    }
  }
  style={Object {}}
/>
`;
